content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,135 @@
+     +- master.key           (used for encrypting the hudson.util.Secret key)
+     +- InstanceIdentity.KEY (used to identity this instance)
+ +- fingerprints   (stores fingerprint records, if any)
+ +- plugins        (root directory for all Jenkins plugins)
+     +- [PLUGINNAME]   (sub directory for each plugin)
+         +- META-INF       (subdirectory for plugin manifest + pom.xml)
+         +- WEB-INF        (subdirectory for plugin jar(s) and licenses.xml)
+     +- [PLUGINNAME].jpi   (.jpi or .hpi file for the plugin)
+ +- jobs           (root directory for all Jenkins jobs)
+     +- [JOBNAME]      (sub directory for each job)
+         +- config.xml     (job configuration file)
+         +- workspace      (working directory for the version control system)
+         +- latest         (symbolic link to the last successful build)
+         +- builds         (stores past build records)
+             +- [BUILD_ID]     (subdirectory for each build)
+                 +- build.xml      (build result summary)
+                 +- log            (log file)
+                 +- changelog.xml  (change log)
+     +- [FOLDERNAME]   (sub directory for each folder)
+         +- config.xml     (folder configuration file)
+         +- jobs           (subdirectory for all nested jobs)
+....
+|===
+
+=== Backup and restore
+link:/doc/book/system-administration/backing-up/[Backing-up/Restoring Jenkins ]
+
+=== Moving/copying/renaming jobs
+You can:
+
+. Move a job from one installation of Jenkins to another by copying the corresponding job directory.
+. Make a copy of an existing job by making a clone of a job directory by a different name.
+. Rename an existing job by renaming a directory. Note that, if you change a job name, you will also need to modify any other job that tries to call the renamed job.
+
+Those operations can be done even when Jenkins is running.
+For changes like these to take effect, you must click "reload config" to force Jenkins to reload configuration from the disk.
+
+==== Batch renaming jobs
+Replacing spaces in job names with underscores.
+[width="100%",cols="100%",]
+|===
+a|
+....
+$ rename 's/\s/_/g' *
+....
+|===
+
+=== Archive unused jobs
+Sometimes you want to remove a job from Jenkins but do so in such a way that you can resurrect it later, if the need arises.
+You can do this by going to `$JENKINS_HOME` and creating an archive of the job directory.
+The following command illustrates how to archive a job 'xyz' and remove it.
+[width="100%",cols="100%",]
+|===
+a|
+....
+$ cd $JENKINS_HOME/jobs
+$ tar czf xyz.tgz xyz
+# go to Jenkins GUI "Manage Jenkins" page and "Reload Configuration from Disk"
+....
+|===
+As long as you are not building the Jenkins.
+You can do this operation without taking Jenkins offline.
+Although you should not build the xyz project while you create an archive.
+See also the link:https://plugins.jenkins.io/shelve-project-plugin/["Shelve Project" plugin.]
```

Comment:
Let's not describe the `tar` process because it doesn't remove the job and does not save any space and does prevent execution of the job.  All it does is a fractional backup.

Use the plugin macro rather than a link
```suggestion
See the plugin:shelve-project["Shelve Project" plugin] for one way of archiving Jenkins jobs.
```